src/actions.py

```diff
@@ -0,0 +1,13 @@
+from .action import Action
```

Comment:
A bit confusing file name, are we going to put all classes inheriting from `Action` here?